src/wast-parser.cc

```diff
@@ -1479,6 +1480,14 @@ Result WastParser::ParsePlainInstr(std::unique_ptr<Expr>* out_expr) {
       break;
     }
 
+    // Simd
+    case TokenType::SimdConst: {
```

Comment:
I think it'd be better to return a TokenType::Const but with a different opcode, like the other const instructions.

Reply:
Yes.  I do wanted to implement v128.const like this way at beginning.

But parse the simd const lists is slightly different to Int32/int64/F32/F64 const.  I'm worried that the changes will mess up the original code.  That's why I use a dedicated token for simd const.

Anyway, I'm re-implementing it and try to keep the code as clean as current.

Reply:
Yeah, I think you'll have to restructure `ParseConst` a bit, but that's OK. Don't worry too much about messing up the code. :-)